keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/optimizer/Optimizer.java

```diff
@@ -35,10 +66,24 @@ static Optimizer of(BayesianNetwork network) {
         }
     }
 
+    /**
+     * Creates a Bayesian network from the given vertices and uses this to
+     * create an {@link Optimizer} which optimizes against the unobserved (latent) variables.
+     *
+     * @param vertices The vertices to create a Bayesian network from.
+     * @return an {@link Optimizer}
+     */
     static Optimizer of(Collection<? extends Vertex> vertices) {
         return of(new BayesianNetwork(vertices));
     }
 
+    /**
+     * Retrieves the connected graph from the given vertex and uses this to
```

Comment:
Maybe `Creates a Bayesian network from the graph connected to the given vertex...` or something like that